src/reporting.js

```diff
@@ -202,7 +197,13 @@ async function _generateReport(console, domain, mediaId, eventContentFile, opts)
     return result;
 }
 
-async function generateResult(console, eventContentFile, filePath, tempDir, script) {
+async function generateResult(console, httpUrl, eventContentFile, filePath, tempDir, script) {
```

Comment:
I'm finding it quite hard to keep track of what the different functions do here (especially the difference between `generateResult` and `generateReport `). `generateResult` == return cached scan or scan file on disk & cache result, `getReport ` == get from cache only, `generateReport ` == download file & run scan it? Might suggest renaming them to not rely on the difference between 'report' and 'result', and adding some doc to this method. Doc on the params would be appreciated too.